sale_order_type_whole_delivered_invoiceability/tests/test_sale_order_type_whole_delivered_invoiceability.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2023 Tecnativa - Pilar Vargas
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests import Form
+
+from odoo.addons.sale_order_whole_delivered_invoiceability.tests import (
+    TestSaleOrderWholeDeliveredInvoiceability,
+)
+
+
+class TestSaleOrderTypeWholeDeliveredInvoiceability(
+    TestSaleOrderWholeDeliveredInvoiceability
+):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.sale_order_type = cls.env["sale.order.type"].create(
+            [
+                {
+                    "name": "Test Normal Order",
+                    "whole_delivered_invoiceability": True,
+                },
+            ]
+        )
+
+    def test_whole_delivered_invoiceability_type(self):
+        with Form(self.order) as order_form:
+            order_form.type_id = self.sale_order_type
+        self.assertTrue(self.order.whole_delivered_invoiceability)
+        self.order.action_confirm()
+        self.assertEqual(self.order.invoice_status, "no")
```

Comment:
This is not something of this module, so don't test it. You are already testing this on the corresponding module (the parent one).

```suggestion
```